src/docs-style-guide.rst

```diff
@@ -277,6 +394,26 @@ Common filler phrases and words include:
 
 This list is not exhaustive. These "canned phrases" are pervasive in technical writing. Remove them whenever they occur.
 
+.. code-block:: python
+  :class: style-checks
+
+  @memoize
```

Comment:
How many of these already exist in the proselint.cliche test? 
I suspect at least some of them.

Reply:
*for all intents and purposes* is the only one. I will remove that from this list.